assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -217,24 +217,24 @@
 
 	--color-wpcom: #{$muriel-blue-500};
 	--color-jetpack: #{$green-jetpack};
-	--color-facebook: #{$color-facebook};
-	--color-twitter: #{$color-twitter};
-	--color-gplus: #{$color-gplus};
-	--color-tumblr: #{$color-tumblr};
-	--color-linkedin: #{$color-linkedin};
-	--color-path: #{$color-path};
-	--color-instagram: #{$color-instagram};
-	--color-eventbrite: #{$color-eventbrite};
-	--color-stumbleupon: #{$color-stumbleupon};
-	--color-reddit: #{$color-reddit};
-	--color-pinterest: #{$color-pinterest};
-	--color-pocket: #{$color-pocket};
-	--color-email: #{$color-email};
-	--color-print: #{$color-print};
-	--color-skype: #{$color-skype};
-	--color-telegram: #{$color-telegram};
-	--color-whatsapp: #{$color-whatsapp};
-	--color-wordpress-org: #{$color-wordpressOrg};
+	--color-facebook: #39579a;
```

Comment:
Feel free to not do this but would it make sense to alphabetically sort this list? At least for me it'd make it easier to find certain variables - I'm coming from a reviewer's perspective but I could imagine that to be useful in general as well.

Here's a sorted list in case you want to use it:

```
--color-email: #f8f8f8;
--color-eventbrite: #ff8000;
--color-facebook: #39579a;
--color-gplus: #df4a32;
--color-instagram: #d93174;
--color-jetpack: #{$green-jetpack};
--color-linkedin: #0976b4;
--color-path: #df3b2f;
--color-pinterest: #cc2127;
--color-pocket: #ee4256;
--color-print: #f8f8f8;
--color-reddit: #5f99cf;
--color-skype: #00aff0;
--color-stumbleupon: #eb4924;
--color-telegram: #0088cc;
--color-tumblr: #35465c;
--color-twitter: #55acee;
--color-whatsapp: #43d854;
--color-wordpress-org: #585c60;
--color-wpcom: #{$muriel-blue-500};
```

Reply:
Agreed, thanks! I've done so for the HEX variables, I'm assuming the others shouldn't since they're both not external brands and don't use HEX yet.